content/docs/reference-events.md

```diff
@@ -149,53 +150,53 @@ boolean shiftKey
 number which
 ```
 
-The `key` property can take any of the values documented in the [DOM Level 3 Events spec](https://www.w3.org/TR/uievents-key/#named-key-attribute-values).
+La propiedad `key` puede tomar cualquiera de los valores documentados en [la especificación de DOM Level 3 Events](https://www.w3.org/TR/uievents-key/#named-key-attribute-values).
 
 * * *
 
-### Focus Events
+### Eventos de Enfoque
 
-Event names:
+Nombres de Eventos:
 
 ```
 onFocus onBlur
 ```
 
-These focus events work on all elements in the React DOM, not just form elements.
+Estos eventos de enfoque funcionan en todos los elementos en React DOM, no solo en los elementos de formulario.
```

Comment:
```suggestion
Estos eventos de enfoque funcionan en todos los elementos en React DOM, no sólo en los elementos de formulario.
```

Reply:
This correction is correct, but the RAE recommends not accentuating `solo` even in ambiguous cases.

Source: https://www.diccionariodedudas.com/solo-o-solo/